pkg/models/usertoken/user_token.go

```diff
@@ -35,3 +36,21 @@ type UserToken struct {
 	RevokedAt     int64
 	UnhashedToken string
 }
+
+const UrgentRotateTime = 1 * time.Minute
```

Comment:
Should me make it configurable ? (In case some servers have poor time sync)

Reply:
Same as above

Reply:
Sure, this PR is already huge